es/index.mjs

```diff
@@ -237,7 +237,7 @@ export default function axiosRetry(axios, defaultOptions) {
 
       config.transformRequest = [(data) => data];
 
-      onRetry(currentState.retryCount, error, config);
+      await onRetry(currentState.retryCount, error, config);
```

Comment:
We also need this! can we merge this feature?

Reply:
+1

Reply:
Hey guys! You can use my forked package, it's also open source, maintained by coinmiles.io and kept up to date with this mirror

https://www.npmjs.com/package/@coin-miles/axios-retry